shell/platform/darwin/common/framework/Source/FlutterStandardCodecHelper.cc

```diff
@@ -210,12 +212,12 @@ void FlutterStandardCodecHelperWriteUTF8(CFMutableDataRef data,
     CFIndex used_length = 0;
     // UTF16 length times 3 will fit all UTF8.
     CFIndex buffer_length = length * 3;
-    UInt8* buffer = (UInt8*)malloc(buffer_length * sizeof(UInt8));
+    std::vector<UInt8> buffer;
```

Comment:
Can you use the constructor that takes a size instead of a separate call to reserve.  I suspect it is as fast or faster and this is highly trafficked code.

Reply:
Haha yes just sent you the same over chat.

Reply:
Actually, it looks like using that constructor might zero out the data.  That's not what we want.  Maybe reserve is better.

Reply:
Tha ctor will zero the buffer at creation time which the current code doesn't do. reserve just makes sure the vector's buffer is sized as requested.